apollo-core/src/main/java/com/ctrip/framework/apollo/core/utils/DeferredLog.java

```diff
@@ -0,0 +1,333 @@
+package com.ctrip.framework.apollo.core.utils;
+
+import org.slf4j.Logger;
+import org.slf4j.Marker;
+
+/**
+ * Only the following methods implement the delayed log function
+ * <pre class="code">
+ *  public void debug(String s, Object... objects);
+ *  public void info(String s, Object... objects);
+ *  public void warn(String s, Object... objects);
+ *  public void error(String s, Throwable throwable);
+ * </pre>
+ * @author kl (http://kailing.pub)
+ * @since 2021/5/19
+ */
+public class DeferredLog implements Logger {
```

Comment:
```suggestion
public class DeferredLogger implements Logger {
```

Reply:
btw, I think we need to add some unit test for this class, or the code coverage rate will drop a lot..